src/pytest_plugins/filler/filler.py

```diff
@@ -196,6 +197,23 @@ def pytest_addoption(parser: pytest.Parser):
         help="Path to dump the transition tool debug output.",
     )
 
+    internal_group = parser.getgroup("internal", "Internal arguments")
+    internal_group.addoption(
+        "--session-temp-folder",
+        action="store",
+        dest="session_temp_folder",
+        type=Path,
+        default=None,
+        help=argparse.SUPPRESS,
+    )
+    evm_group.addoption(
```

Comment:
Should this be `internal_group`?

Reply:
Indeed! thanks!

Reply:
Actually I think it belongs in the `test_group`.